app/controllers/amends/AmendDirectExportPlasticPackagingController.scala

```diff
@@ -49,24 +47,16 @@ class AmendDirectExportPlasticPackagingController @Inject() (
           case None        => form()
           case Some(value) => form().fill(value)
         }
-
-        request.userAnswers.get[TaxReturnObligation](ObligationCacheable) match {
-          case Some(obligation) => Ok(view(preparedForm, obligation))
-          case None             => Redirect(routes.SubmittedReturnsController.onPageLoad())
-        }
+        Ok(view(preparedForm))
```

Comment:
I think we need obligation here just to make sure that the user has gone through the journey (not URL jumping). I asked these same questions in the pass, because the view does not use the obligation, but Linsday pointed out that. Lindsay can you confirm that, please? In this case an alternative may be using fold:

request.userAnswers.get[TaxReturnObligation](ObligationCacheable)
.fold(Redirect(routes.SubmittedReturnsController.onPageLoad())(Ok(view(preparedForm))))

this is valid for all the other changes as they are all the same changes. Just could not be bother to put a comment for each change as are all the same.